c2corg_ui/static/js/imageuploader.js

```diff
@@ -306,7 +306,7 @@ app.ImageUploaderController.prototype.save = function() {
       scope['photo'] = images[i];
       scope['photo']['image_id'] = id;
       scope['photo']['edit_url'] = '/images/edit/' + imageIds[i]['document_id'] + '/' + images[i]['locales'][0]['lang'];
-      scope['photo']['view_url'] = this.url_.buildDocumentUrl('images', imageIds[i]['document_id'], images[i]['locales'][0], images[i]['locales'][0]['lang'] );
+      scope['photo']['view_url'] = this.url_.buildDocumentUrl('images', imageIds[i]['document_id'], images[i]['locales'][0], images[i]['locales'][0]['lang']);
```

Comment:
That's just that this change was already done in master and thus there is no longer a difference between what you did and master.